Makefile

```diff
@@ -206,6 +206,9 @@ push: push-datamgr push-plugin push-backup-driver
 push-pp:
 	$(MAKE) push-plugin LOCALMODE=true VERSION=$(VERSION)-pp
 
+push-pp-wcp:
+	$(MAKE) push-plugin push-backup-driver LOCALMODE=true VERSION=$(VERSION)-pp-wcp
```

Comment:
Should we just update this to the following so the the flag is set for data-manager as well?
push-pp:
	$(MAKE) push LOCALMODE=true VERSION=$(VERSION)-pp

Reply:
We don't have to push data-manager image since PowerProtect don't use data-manager at all.